Add docs/routing/hints.mdx (#1033)
pg-sharding/spqr: docs/routing/hints.mdx

 t                 
(1 row)
```

### `default_route_behaviour`

Whether to explicitly block multishard queries. Overload for [query_routing.default_route_behaviour](/configuration/router#feature-flags).

```sql
DROP DATABASE prod;
ERROR:  client processing error: error processing query 'DROP DATABASE prod': failed to match any datashard., tx status IDLE

DROP DATABASE prod /* __spqr__default_route_behaviour: allow */;
NOTICE: send query to shard(s) : shard1,shard2,shard3,shard4
```

### `__spqr__execute_on`

The __spqr__execute_on parameter is used to specify the shard on which a query should be executed.

```sql
SELECT col1 FROM xx ORDER BY col1 /* __spqr__execute_on: shard2 */;
```

### `__spqr__reply_notice`

Whether to show notice messages. Overload for [query_routing.default_route_behaviour](/configuration/router#debug-settings). 

```sql
SET __spqr__reply_notice TO true;
INSERT INTO xxtest_sw (id) VALUES(1), (2), (3);
NOTICE: send query to shard(s) : sh1
```

### `__spqr__sharding_key`

The __spqr__sharding_key explicitly pass the sharding key to the SPQR Router.

```sql
INSERT INTO test(id, age) VALUES (10, 16) /*__spqr__sharding_key: 30*/;
```

### `__spqr__distribution`

> The __spqr__distribution explicitly specify which distribution should you the SPQR router to the query.

Author: ```suggestion
The __spqr__distribution explicitly specify which distribution the SPQR router should use for query routing.
```
